models/user.go

```diff
@@ -330,27 +332,27 @@ func FindUserByConfirmationToken(tx *storage.Connection, token string) (*User, e
 
 // FindUserByEmailAndAudience finds a user with the matching email and audience.
 func FindUserByEmailAndAudience(tx *storage.Connection, email, aud string) (*User, error) {
-	return findUser(tx, "instance_id = ? and LOWER(email) = ? and aud = ?", uuid.Nil, strings.ToLower(email), aud)
+	return findUser(tx, "instance_id = ? and LOWER(email) = ? and aud = ? and is_sso_user = false", uuid.Nil, strings.ToLower(email), aud)
```

Comment:
hmm why don't we pass `is_sso_user` as a param in the function instead of hardcoding it to false here? does this mean that for sso, we won't need to reuse these functions?

Reply:
Because you don't want to actually find SSO users whenever this is used. SSO users are only "accessible" via the SAML endpoint, and can't be "found" in sign up, OTP, sign in or other such queries.

Reply:
Yeah, it makes sense to now show SSO users though I kind of see where Kang is coming from - from the name alone I would expect the method to return all users unless otherwise specified. Guess it's fine as long as we make it standardised across all the user methods